README.md

```diff
@@ -1,67 +1,37 @@
-# packer.nvim
```

Comment:
Instead of keeping the old one around. Would it be better to simply link to the `master` version at the top of the `main` version?

Reply:
Either is basically equivalent to me, because - once `main` becomes the canonical `packer` - any duplication (link or copy) will need to be removed anyway. If you're concerned about copying a version of the README that becomes stale, then sure, a link is fine. I don't foresee many changes to the README before `main` lands, though.